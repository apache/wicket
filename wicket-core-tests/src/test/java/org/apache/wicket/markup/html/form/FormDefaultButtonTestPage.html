<html>
<body>
	<form wicket:id="form">
		<button wicket:id="default"></button>
	</form>
</body>
</html>
