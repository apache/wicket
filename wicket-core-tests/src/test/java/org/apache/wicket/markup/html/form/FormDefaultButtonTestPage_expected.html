<html>
<head><script type="text/javascript" src="../resource/org.apache.wicket.resource.JQueryResourceReference/jquery/jquery-3.7.1.js"></script>
<script type="text/javascript" src="../resource/org.apache.wicket.ajax.AbstractDefaultAjaxBehavior/res/js/wicket-ajax-jquery.js"></script>
<script type="text/javascript">
/*<![CDATA[*/
Wicket.Event.add(window, "domready", function(event) { 
Wicket.Event.add('form2', 'keypress', function(event) { if (event.target.tagName.toLowerCase() !== 'input' || event.which != 13) return;var b = document.getElementById('default1');if (window.getComputedStyle(b).visibility === 'hidden') return;event.stopPropagation();event.preventDefault();if (b != null && b.onclick != null && typeof (b.onclick) != 'undefined') {var r = Wicket.bind(b.onclick, b)();if (r != false) b.click();} else {b.click();}return false;;});;
Wicket.Event.publish(Wicket.Event.Topic.AJAX_HANDLERS_BOUND);
;});
/*]]>*/
</script>
</head><body>
	<form wicket:id="form" id="form2" method="post" action="./org.apache.wicket.markup.html.form.FormDefaultButtonTestPage?0-1.-form"><div id="form2_hf_0" hidden="" class="hidden-fields"></div>
		<button wicket:id="default" name="default" id="default1"></button>
	</form>
</body>
</html>
